carma-messenger-ui/website/widgets/emergencyResponse/widget.js

```diff
@@ -35,10 +51,31 @@ var subscribe_bsm = () => {
         name: '/bsm_outbound',
         messageType: 'cav_msgs/BSM'
     });
-
     listenerBSM.subscribe(function (message) {
         if (message.core_data != undefined && message.core_data.latitude != undefined && message.core_data.longitude != undefined) {
+            //Vehicle current location
             $("#positionValue").text(message.core_data.latitude + "," + message.core_data.longitude);
+            data.features = [];
```

Comment:
Since the ERV will be publishing BSMs at 10 Hz, do you have any idea whether this update (which will typically just affect the first point in `data.features` since this will be the ERV's current location) rate will cause the map display to lag? Maybe it's nothing to worry about, but was just curious whether you had thoughts on it

Reply:
I do not think so. I test it with local integration testing with message pub at 10HZ and do not see a lag.

Reply:
The destination points will be reduced as the vehicle move toward to destination as well right? so I think it is safe to use the latest BSM message at all time.